_docs/11-app-config.md

```diff
@@ -65,6 +66,7 @@ The noteworthy parameters here are:
 - **servicesConfiguration:** Object that contains the services configuration parameters. Learn more [here](#services-configuration).
 - **appMenuConfiguration:** Object defining application menu structure. Learn more [here](#menu-configuration).
 - **applicationLocales:** Set of available locales for the application.
+- **defaultLocale:** Default language if browser language is not found in application.
```

Comment:
Please make the appointment that it is necessary that this value must exist into _applicationLocales_